.github/workflows/benchmark.yml

```
@@ -13,8 +13,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+        uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
         with:
           python-version: '3.11'
```

```suggestion
          python-version: '3.12'
```
It seems ASV support Python 3.12 since version 0.6.2.